src/System.Console/ref/System.Console.cs

```diff
@@ -36,8 +36,11 @@ public static partial class Console
         public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop, char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor) { }
         public static bool NumberLock { get { return default(bool); }}
```

Comment:
@weshaggard or @ericstj, did i update the contract version stuff correctly? Thanks.